.github/workflows/ci.yaml

```diff
@@ -80,8 +82,10 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.toolchain }}
+          components: ${{ matrix.components }}
           targets: wasm32-unknown-unknown
-      - run: cargo build --verbose --target wasm32-unknown-unknown ${{ matrix.cargoflags }}
+      - uses: jetli/wasm-pack-action@v0.4.0
```

Comment:
I'm wary of adding 3rd-party github actions -- can we just use the official [installer](https://rustwasm.github.io/wasm-pack/installer/) like I did for `wasmtime` below?

Reply:
This is a reasonably popular action, but, sure, we can use official installer too, especially since wasm-pack provides prebuilt binaries and doesn't need recompilation from source.